rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -287,29 +287,6 @@ public String defineNodeSource(final String sessionId, final String nodeSourceNa
                                                                                              nodesRecoverable));
     }
 
-    /*
-     * (non-Javadoc)
-     * 
-     * @see
-     * org.ow2.proactive_grid_cloud_portal.rm.client.RMService#createNodeSource(java.lang.String,
-     * java.lang.String, java.lang.String, java.lang.String[], java.lang.String[], java.lang.String,
-     * java.lang.String[], java.lang.String[])
-     */
-    public String createNodeSource(final String sessionId, final String nodeSourceName, final String infrastructureType,
-            final String[] infrastructureParameters, final String[] infrastructureFileParameters,
-            final String policyType, final String[] policyParameters, final String[] policyFileParameters,
-            final String nodesRecoverable) throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.createnodeSource(sessionId,
-                                                                                             nodeSourceName,
-                                                                                             infrastructureType,
-                                                                                             infrastructureParameters,
-                                                                                             infrastructureFileParameters,
-                                                                                             policyType,
-                                                                                             policyParameters,
-                                                                                             policyFileParameters,
-                                                                                             nodesRecoverable));
-    }
-
     @Override
     public String deployNodeSource(final String sessionId, final String nodeSourceName)
             throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)